### PR TITLE
Docs for getting started

### PR DESCRIPTION
**Pull request by perolavsvendsen**
Amendment to #239 - a file was missing.

### CODE REVIEW

File: docs/preparations.rst [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+/scratch/<case_directory>/share/metadata/fmu_results.yml. The case metadata are read by
+individual export jobs, and, if you opt to upload data into Sumo, the case metadata are
+used to register the case.
+
+Case metadata are made by a hooked ERT workflow running ``PRE_SIMULATION``.
+
+To make this, first create the workflow file in ``ert/bin/workflows/xhook_create_case_metadata``.
+
+.. note::
+    The "xhoox" prefix is convention, but not mandatory. As all workflows will be included in the
```
Comment (jcrivenaes) [inline docs/preparations.rst:191]: xhoox -> xhook

File: docs/preparations.rst [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+-------------------------
+
+To verify that data export now works, add one job to your workflow. Pick something simple,
+such as depth surfaces from the structural model or similar. Use one of the examples on
+the next page to get going, and/or have a look at the Drogon tutorial project.
+
+**What about Sumo**
+Odds are that you are implementing rich metadata export so that you can start utilizing
+Sumo. Producing metadata with exported data is a pre-requisite for using Sumo. When you
+have undertaken the steps above, you are good to go! Head to the Sumo documentation to
```
Comment (jcrivenaes) [inline docs/preparations.rst:255]: Hyperlink to SUMO?

File: docs/preparations.rst [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+get going 👍
```
Comment (jcrivenaes) [inline docs/preparations.rst:256]: Hyper link to SUMO?

**Review by jcrivenaes — APPROVED** [review]
Nice  introduction

File: docs/preparations.rst (hunk 3 above)
Comment (perolavsvendsen) [reply docs/preparations.rst]: Done

**Pull request merged**